workalendar/tests/test_europe.py

```diff
@@ -493,6 +494,34 @@ def test_year_2013(self):
         self.assertIn(date(2013, 12, 26), holidays)  # San Stefano
 
 
+class LatviaTest(GenericCalendarTest):
+
+    cal_class = Latvia
+
+    def test_year_2017(self):
+        """
+        https://www.bank.lv/en/about-us/public-holidays-in-latvia
+        """
+        holidays = self.cal.holidays_set(2017)
+        self.assertIn(date(2017,  1,  1), holidays)  # New Year's Day
```

Comment:
It's fine because the flake8 test don't check the test files, but multiple spaces are against the pep8 rules.